packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2506,55 +2492,132 @@ class RenderPointerListener extends RenderProxyBoxWithHitTestBehavior {
   /// Called when a pointer that triggered an [onPointerDown] changes position.
   PointerMoveEventListener onPointerMove;
 
+  /// Called when a pointer that triggered an [onPointerDown] is no longer in
+  /// contact with the screen.
+  PointerUpEventListener onPointerUp;
+
+  /// Called when the input from a pointer that triggered an [onPointerDown] is
+  /// no longer directed towards this receiver.
+  PointerCancelEventListener onPointerCancel;
+
+  /// Called when a pointer signal occurs over this object.
+  PointerSignalEventListener onPointerSignal;
+
+  @override
+  void performResize() {
+    size = constraints.biggest;
+  }
+
+  @override
+  void handleEvent(PointerEvent event, HitTestEntry entry) {
+    assert(debugHandleEvent(event, entry));
+    // The onEnter, onHover, and onExit events are triggered from within
+    // MouseTracker, not here.
+    if (onPointerDown != null && event is PointerDownEvent)
+      return onPointerDown(event);
+    if (onPointerMove != null && event is PointerMoveEvent)
+      return onPointerMove(event);
+    if (onPointerUp != null && event is PointerUpEvent)
+      return onPointerUp(event);
+    if (onPointerCancel != null && event is PointerCancelEvent)
+      return onPointerCancel(event);
+    if (onPointerSignal != null && event is PointerSignalEvent)
+      return onPointerSignal(event);
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    final List<String> listeners = <String>[];
+    if (onPointerDown != null)
+      listeners.add('down');
+    if (onPointerMove != null)
+      listeners.add('move');
+    if (onPointerUp != null)
+      listeners.add('up');
+    if (onPointerCancel != null)
+      listeners.add('cancel');
+    if (onPointerSignal != null)
+      listeners.add('signal');
+    if (listeners.isEmpty)
+      listeners.add('<none>');
+    properties.add(IterableProperty<String>('listeners', listeners));
+    // TODO(jacobr): add raw listeners to the diagnostics data.
+  }
+}
+
+/// Calls callbacks in response to events of mouse pointers, including
+/// [onEnter], [onHover], and [onExit].
+///
+/// If it has a child, defers to the child for sizing behavior.
+///
+/// If it does not have a child, grows to fit the parent-provided constraints.
+///
+/// See also:
+///
+///  * [RenderPointerListener], which contains events of all kinds of pointers,
+///    including [onPointerDown], [onPointerUp], and [onPointerMove].
+class RenderMouseListener extends RenderProxyBox {
+  /// Creates a render object that forwards pointer events to callbacks.
+  ///
+  /// The [behavior] argument defaults to [HitTestBehavior.deferToChild].
```

Comment:
This is no behavior parameter in this class?

Reply:
Thanks for pointing it out.